s/elements/stickpad/element.ts

```diff
@@ -81,6 +81,14 @@ export class NubStickpad extends LitElement {
 		},
 	}
 
+	connectedCallback() {
```

Comment:
maybe instead of connectedCallback, can we do something more terse? maybe like:
```js
#resize_observer = new ResizeObserver(() => this.#reset_offset_to_center()).observe(this)
```

Reply:
where do we then call this private method?

Reply:
hmm, it actually doesn't need to be called at all -- it is misleading in that way.

maybe it's best in the `firstUpdated` call, just this one-liner:
```js
new ResizeObserver(() => this.#reset_offset_to_center()).observe(this)
```

Reply:
it's kind of hard to read, so maybe we make a helper util so it reads more like
```js
on_element_resize(this, () => this.#reset_offset_to_center)
```